lib/streamlit/server/server.py

```diff
@@ -94,6 +100,10 @@
 UNIX_SOCKET_PREFIX = "unix://"
 
 
+# Wait for the script run result for 60s and if no result is available give up
+SCRIPT_RUN_CHECK_TIMEOUT = 60
```

Comment:
I think this would be better as a a config option. wdyt?

Reply:
This is ultimately a product decision, I think - so cc'ing @asaini for this one. (My instinct would be to leave it hardcoded for now, and then promote it to a config option if there's demand from users - but that may be shortsighted if we think that people will immediately be unsatisfied with the default?)

Reply:
I think it's also probably more of a product decision, but my hunch is that it doesn't matter too much what we end up doing for now as the only user of this route will be our own services. It should also be easy to promote to be a proper config option should we see the need to do so.

Reply:
Hi All - Can you please help me understand the consequence of this decision? 
- Why would users want to be able to control this via a config option? 
- What happens if there is no result after 60s? - do we assume that the app is dead? 
- Do we expect any impact on user experience whether this number is 5s or 60s? 

Is there a spec which talks about how this health check works? Would love to see a high level architectural diagram to better understand